rpcs3/Emu/RSX/Overlays/overlay_edit_text.cpp

```diff
@@ -172,7 +205,7 @@ namespace rsx
 				{
 					// TODO: Scrolling by using scroll offset
 					cmd.config.clip_region = true;
-					cmd.config.clip_rect   = {f32(x), f32(y), f32(x + w), f32(y + h)};
+					cmd.config.clip_rect   = {static_cast<f32>(x), static_cast<f32>(y), static_cast<f32>(x + w), static_cast<f32>(y + h)};
```

Comment:
There's a lot of this static cast style change going around, let's have it as a separate commit.